### ISSUE TITLE
Use gsn from TypeScript

### ISSUE THREAD

**Issue by vrde** [opengsn/gsn#434]
Hello, I'm trying to use `v0.9.0` from TypeScript, and I get this error:

```
node_modules/@opengsn/gsn/dist/src/relayclient/ContractInteractor.d.ts:6:130 - error TS2307: Cannot find module '../../types/truffle-contracts' or its corresponding type declarations.

6 import { IPaymasterInstance, IRelayHubInstance, IRelayRecipientInstance, IStakeManagerInstance, ITrustedForwarderInstance } from '../../types/truffle-contracts';
                                                                                                                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

It looks like a bug on your side, thoughts?

**Comment by vrde** [comment]
I see the directory is explicitly ignored

https://github.com/opengsn/gsn/blob/5d8ef47144efc042c8037ddc0ef5b1eedd684674/.gitignore#L18

**Comment by drortirosh** [comment]
sorry for the late response...
I think this issue is solved with the latest version (2.0.0-beta)
